src/mpid/ch4/shm/ipc/src/globals.c

```diff
@@ -6,8 +6,44 @@
 #include "mpidimpl.h"
 #include "ipc_types.h"
 
+/*
+=== BEGIN_MPI_T_CVAR_INFO_BLOCK ===
+cvars:
+    - name        : MPIR_CVAR_CH4_IPC_MAP_REPEAT_ADDR
+      category    : CH4
+      type        : boolean
+      default     : true
+      class       : none
+      verbosity   : MPI_T_VERBOSITY_USER_BASIC
+      scope       : MPI_T_SCOPE_ALL_EQ
+      description : >-
+        If an address is used more than once in the last ten send operations,
+        map it for IPC use even if it is below the IPC threshold.
```

Comment:
should update this to reflect it multiple uses now

Reply:
Updated both.